code/modules/client/client_procs.dm

```diff
@@ -1082,6 +1082,9 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 				if(OOC_CHANNEL)
 					var/ooc = tgui_say_create_open_command(OOC_CHANNEL)
 					winset(src, "default-[REF(key)]", "parent=default;name=[key];command=[ooc]")
+				if(ADMIN_CHANNEL)
```

Comment:
can non-admins call this

Reply:
that's a client proc and it's filtered by /datum/keybinding/admin/can_use

server filtering happens at /datum/tgui_say/proc/delegate_speech

Reply:
wait no this is getting through, probably because of the weird way tgui say is hooked up to the binds, gimme a sec

Reply:
sorted